db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -246,16 +246,24 @@ Status DBImpl::FlushMemTableToOutputFile(
       // be pessimistic and try write to a new MANIFEST.
       // TODO: distinguish between MANIFEST write and CURRENT renaming
```

Comment:
If it is convenient, you can also add  `.PermitUncheckedError()` to the remaining SetBGError at about L1193 and about L3000

https://github.com/facebook/rocksdb/pull/7342/files